src/nupic/support/__init__.py

```diff
@@ -431,13 +431,18 @@ def makeKey(name):
     replacements[makeKey('SYSLOG_HANDLER_ADDRESS')] = '"/var/run/syslog"'
   elif platform.startswith('linux'):
     replacements[makeKey('SYSLOG_HANDLER_ADDRESS')] = '"/dev/log"'
+  elif platform.startswith('win'):
+    replacements[makeKey('SYSLOG_HANDLER_ADDRESS')] = '"log"'
```

Comment:
Sorry, a late in the day reply. My comment referred to the nupic.log (file handler one). A lot of temporary files, such as those produced via py.test runs, get placed into a Users hidden AppData temp directory (with semi-random directory names to avoid clashes). We'll have a look at the sys log handling, in another PR.